test/test_nn.py

```diff
@@ -15884,8 +15884,6 @@ def forward(self, x):
         # Test meta module instantiation.
         input = torch.randn(5, 10, device=device, dtype=dtype)
         m = MyModule(10, 1, device='meta', dtype=dtype)
-        with self.assertRaises(NotImplementedError):
-            m(input)
```

Comment:
The changes here look a little questionable. I think we *want* it to be possible to stream inputs through modules, in general, so you should keep calling the `m(input)` (and maybe put some asserts on them). So instead of removing the `m(input)` call entirely, if they are now passing, you should keep calling them to exercise.